administrator/language/de-DE/com_workflow.ini

```
@@ -47,7 +47,7 @@ COM_WORKFLOW_MODIFIED_AT_ASC="Bearbeitungsdatum aufsteigend"
 COM_WORKFLOW_MODIFIED_AT_DESC="Bearbeitungsdatum absteigend"
 COM_WORKFLOW_MODIFIED_LABEL="Bearbeitungsdatum"
 COM_WORKFLOW_MSG_DELETE_IS_DEFAULT="Der Standard-Eintrag kann nicht gelöscht werden."
-COM_WORKFLOW_MSG_DELETE_STAGE_IS_ASSIGNED="Diese Phase wird von der Komponente verwendet."
+COM_WORKFLOW_MSG_DELETE_STAGE_IS_ASSIGNED="Diese Phase wird einem Element dieser Komponente verwendet und muss daher veröffentlicht bleiben."
```

```suggestion
COM_WORKFLOW_MSG_DELETE_STAGE_IS_ASSIGNED="Diese Phase wird von einem Element dieser Komponente verwendet und muss daher veröffentlicht bleiben."
```
Nur das klingt etwas _holprig_, war aber im Original PR #2673 schon so.